uint8 uav_number
uint8 target_number
bool able_to_take_off
bool before_rect_success
bool cross_rect_success
bool room_arrange_success